guide/popular-topics/embeds.md

```diff
@@ -399,7 +399,7 @@ If you want to build the new embed data on a previously sent embed template, mak
 There are a few limits to be aware of while planning your embeds due to the API's limitations. Here is a quick reference you can come back to:
 
 - Embed titles are limited to 256 characters
-- Embed descriptions are limited to 2048 characters
+- Embed descriptions are limited to 4096 characters
```

Comment:
This is already in the V13 guide.

See [here](https://discordjs.guide/popular-topics/embeds.html#embed-limits)